### PR TITLE
Add canViewJudgeOutputs flag to edit assignment form

### PR DESCRIPTION
**Pull request by Neloop**

### CODE REVIEW

File: src/components/forms/EditAssignmentForm/EditAssignmentForm.js [hunk 1]
```diff
@@ -407,7 +411,7 @@ class EditAssignmentForm extends Component {
 
           <Grid fluid>
             <Row>
-              <Col md={5}>
+              <Col md={6}>
```
Comment (krulis-martin) [inline src/components/forms/EditAssignmentForm/EditAssignmentForm.js]: Do not touch my carefully calibrated col sizes.

File: src/components/forms/EditAssignmentForm/EditAssignmentForm.js [hunk 2]
```diff
@@ -420,7 +424,20 @@ class EditAssignmentForm extends Component {
                   }
                 />
               </Col>
-              <Col md={7}>
+              <Col md={6}>
```
Comment (krulis-martin) [inline src/components/forms/EditAssignmentForm/EditAssignmentForm.js]: Do not touch my carefully calibrated col sizes.

File: src/locales/cs.json [hunk 3]
```diff
@@ -161,6 +161,7 @@
   "app.editAssignment.title": "Upravit zadání úlohy",
   "app.editAssignment.validation.versionDiffers": "Někdo změnil tuto zadanou úlohu v průběhu její editace. Prosíme obnovte tuto stránku a proveďte své změny znovu.",
   "app.editAssignmentForm.allowSecondDeadline": "Povolit druhý termín odevzdání",
+  "app.editAssignmentForm.canViewJudgeOutputs": "Viditelnost výstupů ze sudího",
```
Comment (krulis-martin) [inline src/locales/cs.json]: Viditelnost logu sudího

File: src/components/forms/EditAssignmentForm/EditAssignmentForm.js (hunk 1 above)
Comment (Neloop) [reply src/components/forms/EditAssignmentForm/EditAssignmentForm.js]: Too bad, this is better...

**Pull request merged**